public/app/features/trails/helpers/MetricDatasourceHelper.ts

```diff
@@ -4,8 +4,12 @@ import {
   DataSourceGetTagValuesOptions,
   MetricFindValue,
 } from '@grafana/data';
-import { PrometheusDatasource, PromMetricsMetadata, PromMetricsMetadataItem, PromQuery } from '@grafana/prometheus';
-import PromQlLanguageProvider from '@grafana/prometheus/src/language_provider';
+import PromQlLanguageProvider, {
+  PrometheusDatasource,
+  PromMetricsMetadata,
+  PromMetricsMetadataItem,
+  PromQuery,
+} from '@grafana/prometheus';
```

Comment:
If you take the `packages/grafana-prometheus/src/index.ts` suggestion, this becomes:
```suggestion
import {
  PrometheusDatasource,
  PromMetricsMetadata,
  PromMetricsMetadataItem,
  PromQlLanguageProvider,
  PromQuery,
} from '@grafana/prometheus';
```